library/res/layout-sw600dp/app_passcode_keyboard.xml

```diff
@@ -1,78 +1,62 @@
         android:id="@+id/AppUnlockLinearLayout1"
-        android:layout_width="wrap_content"
+        android:layout_width="500dp"
         android:layout_height="wrap_content"
         android:layout_gravity="center"
         android:gravity="center"
         android:orientation="vertical"
-        android:padding="16dp"
-        android:maxWidth="320dp">
-
-    <LinearLayout android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:orientation="horizontal">
+        android:paddingLeft="16dp"
+        android:paddingRight="16dp">
 
         <ImageView
             android:id="@+id/passcode_logo"
-            android:layout_width="32dp"
-            android:layout_height="32dp"
+            android:contentDescription="@string/cd_passcodelock_logo"
+            android:layout_width="100dp"
+            android:layout_height="100dp"
+            android:paddingBottom="16dp"
+            android:paddingTop="16dp"
             android:scaleType="centerInside"
             android:src="@drawable/passcode_logo" >
         </ImageView>
 
         <TextView
-            android:id="@+id/top_message"
+            android:id="@+id/passcodelock_prompt"
             android:layout_width="fill_parent"
             android:layout_height="wrap_content"
-            android:layout_gravity="center_vertical"
-            android:layout_marginLeft="8dp"
-            android:text="@string/passcode_enter_passcode"
+            android:gravity="center"
+            android:paddingBottom="16dp"
+            android:paddingTop="16dp"
+            android:text="@string/passcodelock_prompt_message"
             android:textAlignment="center"
-            android:textColor="#FF464646"
+            android:textColor="@color/passcodelock_prompt_text_color"
             android:textSize="18sp" />
-    </LinearLayout>
 
-        <LinearLayout
-            android:id="@+id/pin_code_row"
+        <EditText
+            android:id="@+id/pin_field"
+            style="@style/PasscodeEditTextStyle"
+            android:gravity="center"
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
-            android:orientation="horizontal" >
-
-            <EditText
-                android:id="@+id/pincode_1"
-                style="@style/PasscodeInputItemStyle" >
-
-                <requestFocus />
-            </EditText>
-
-            <EditText
-                android:id="@+id/pincode_2"
-                style="@style/PasscodeInputItemStyle" >
-            </EditText>
-
-            <EditText
-                android:id="@+id/pincode_3"
-                style="@style/PasscodeInputItemStyle" >
-            </EditText>
-
-            <EditText
-                android:id="@+id/pincode_4"
-                style="@style/PasscodeInputItemStyle" >
-            </EditText>
-        </LinearLayout>
+            android:layout_marginRight="38dp"
+            android:hint="@string/passcodelock_hint"
```

Comment:
Do you think we even need the hint? It says 'Enter PIN code' already right above it.